### PR TITLE
Standard Transformer Config Validation

### PR DESCRIPTION
**Pull request by pradithya**
**What this PR does / why we need it**:

Add standard transformer config validation by verifying the necessary field and crosscheck with feature/entity in feast. 

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [X] Added unit test, integration, and/or e2e tests
- [ ] ~~Tested locally~~
- [ ] ~~Updated documentation~~
- [ ] ~~Update Swagger spec if the PR introduce API changes~~
- [ ] ~~Regenerated Golang and Python client if the PR introduce API changes~~

### CODE REVIEW

File: api/api/version_endpoints_api.go [hunk 1]
```diff
@@ -164,9 +166,15 @@ func (c *EndpointsController) CreateEndpoint(r *http.Request, vars map[string]st
 
 	// validate transformer
 	if newEndpoint.Transformer != nil {
-		err := validateTransformer(newEndpoint.Transformer)
+		err := c.validateTransformer(r.Context(), newEndpoint.Transformer)
```
Comment (ariefrahmansyah) [inline api/api/version_endpoints_api.go:169]: we already have ctx object above. i think we can reuse it

File: api/api/version_endpoints_api.go [hunk 2]
```diff
@@ -223,9 +231,15 @@ func (c *EndpointsController) UpdateEndpoint(r *http.Request, vars map[string]st
 	if newEndpoint.Status == models.EndpointRunning || newEndpoint.Status == models.EndpointServing {
 		// validate transformer
 		if newEndpoint.Transformer != nil {
-			err := validateTransformer(newEndpoint.Transformer)
+			err := c.validateTransformer(r.Context(), newEndpoint.Transformer)
```
Comment (ariefrahmansyah) [inline api/api/version_endpoints_api.go:234]: same as above. we can reuse existing `ctx`

File: api/pkg/transformer/feast/validator.go [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+func ValidateTransformerConfig(ctx context.Context, coreClient core.CoreServiceClient, trfCfg *transformer.StandardTransformerConfig) error {
+	if trfCfg.TransformerConfig == nil {
+		return NewValidationError("transformerConfig is empty")
+	}
+
+	if len(trfCfg.TransformerConfig.Feast) == 0 {
+		return NewValidationError("feature retrieval config is empty")
+	}
+
+	req := &core.ListEntitiesRequest{}
```
Comment (ariefrahmansyah) [inline api/pkg/transformer/feast/validator.go:23]: nitpick: we can specify filter options: `https://github.com/feast-dev/feast/blob/v0.8.2/sdk/go/protos/feast/core/CoreService.pb.go#L2331-L2343`

**Review by ariefrahmansyah — APPROVED** [review]
lgtm (with some nitpick comments)

File: api/pkg/transformer/feast/validator.go (hunk 3 above)
Comment (pradithya) [reply api/pkg/transformer/feast/validator.go:23]: I want to get all entities in one call, thus not using filter

**Pull request merged**